testsuite/tests/test/conditional-tests/test.py

```diff
@@ -0,0 +1,37 @@
+"""
+Check that conditional tests can be properly processed and exported
+"""
+
+from drivers.alr import alr_manifest, init_local_crate, init_local_crate, run_alr
+from drivers.helpers import append_to_file
+from drivers.asserts import assert_substring
+
+init_local_crate(with_test=False)
+
+# Add a conditional test
+append_to_file(alr_manifest(),
+               [
+                   "[[test]]",
+                   "[test.'case(os)'.'...']",
```

Comment:
From how I understand the TOML format, shouldn't a dynamic property just have the `[test.'case(os)'...]` line, without the `[[test]]`?

Reply:
Both should work. I'll add it to the test.

Reply:
The reason I only used the array form is because I remember when I did dynamic actions that was the problematic one, so I instinctively went directly for it.